docker/entrypoint.sh

```diff
@@ -33,7 +33,9 @@ case "$1" in
         elif [[ "$TEST_LWD_TRANSACTIONS" -eq "1" ]]; then
             # Starting at a cached tip, test a gRPC call to lightwalletd, which calls Zebra.
             exec cargo "test" "--locked" "--release" "--features" "lightwalletd-grpc-tests" "--test" "acceptance" "--" "--nocapture" "--ignored" "sending_transactions_using_lightwalletd"
-
+        elif [[ "$TEST_LWD_FULL_SYNC" -eq "1" ]]; then
+            # TODO: add relevant comment
+            exec cargo "test" "--locked" "--release" "--features" "lightwalletd-grpc-tests" "--test" "acceptance" "--" "--nocapture" "--ignored" "sending_transactions_using_lightwalletd"
```

Comment:
Minor: This code is never used, I moved the `TEST_LWD_FULL_SYNC` case to the end of the section above.